3506/2/Gemfile

```diff
@@ -0,0 +1,7 @@
+source 'https://rubygems.org'
+
+gem 'haversine'
```

Comment:
Please lock versions of your gems. It will save you from app break in future

Reply:
Gems are locked in Gemfile.lock and it's commited into repository. Even if gem updated in the future, bundle install will install the version specified in Gemfile.lock